nvdaHelper/vbufBackends/mshtml/mshtml.cpp

```diff
@@ -1007,6 +1010,13 @@ if(!(formatState&FORMATSTATE_INSERTED)&&nodeName.compare(L"INS")==0) {
 	if(nodeName.compare(L"OL")==0||nodeName.compare(L"UL")==0) {
 		//Ordered lists should number their list items
 		LIIndex=1;
+		// set the list index if list has start attribute
+		if ((tempIter = attribsMap.find(L"HTMLAttrib::start")) != attribsMap.end()) {
+			if (!tempIter->second.empty()) {
+				LIIndex = stoi(tempIter->second);

```

Comment:
This could throw either `std::invalid_argument` or `std::out_of_range` check https://en.cppreference.com/w/cpp/string/basic_string/stol. Please handle this.